frontend/public/components/_project-overview.scss

```diff
@@ -3,29 +3,38 @@
 }
 
 .project-overview__group-heading {
-  border-bottom: 1px solid black;
+  border-bottom: 1px solid $color-pf-black-400;
   margin: 0;
   padding: 5px 0;
+  font-size: 22px;
+  font-weight: 300;
+  line-height: normal;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
 }
 
 .project-overview__list {
   margin-top: 0;
 }
 
 .project-overview__item-heading {
+  font-size: 16px;
   margin: 0;
 }
 
 .project-overview__additional-info {
   align-items: center;
   display: grid;
+  grid-column-gap: 10px;
   grid-template-columns: 50% 50%;
   grid-template-areas: "controller status";
   width: 100%;
 }
 
 .project-overview__detail--controller {
   grid-area: controller;
+  min-width: 0;
```

Comment:
This could use a comment explaining why it's necessary.

Reply:
Needed to enable `text-overflow: ellipsis" within a css-grid area. I'll add comment.